extension-androidauto/src/main/java/com/mapbox/maps/extension/androidauto/DefaultMapboxCarMapGestureHandler.kt

```diff
@@ -94,6 +94,18 @@ open class DefaultMapboxCarMapGestureHandler : MapboxCarMapGestureHandler {
     }
   }
 
+  /**
+   * TODO docs
+   */
+  override fun onClick(
+    mapboxCarMapSurface: MapboxCarMapSurface,
+    x: Float,
+    y: Float
+  ) {
+    logI(TAG, "click $x, $y")
+    // TODO figure out who should handle this event
```

Comment:
@kmadsen would you mind sharing some info about how mapbox should be handling this?

Reply:
This is an `open class` so it is not required for the `DefaultMapboxCarMapGestureHandler` to implement this

go ahead and delete it